dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/AI/OpenAI/OpenAIClient.cs

```diff
@@ -8,7 +8,9 @@
 using Microsoft.TeamsAI.Exceptions;
 using Microsoft.TeamsAI.Utilities;
 
+#pragma warning disable IDE0130 // Namespace does not match folder structure
 namespace Microsoft.TeamsAI.OpenAI
```

Comment:
How about just using `Microsoft.TeamsAI.AI.OpenAI`?

Reply:
Just realized that changing this namespace will not affect samples. Updated.